### PR TITLE
Fix new course form

### PR DESCRIPTION
**Pull request by andrewjknapp**
# Bring styles on new course form in line with Figma

## Goals

Update page 1 and 3 of the New Course Form to match the order and styling found in the Figma.

## Updates
- Form inputs are placed in the correct order
- Added inputs for Course Link and Google Classroom Class Code
- Updated cache upon course creation for the Registration and Course Management Cache.

**Link to [figma](https://www.figma.com/file/RqyiCN4499Tv1AokoVTqwv/Omou-User-Flows?node-id=395%3A39669)**

### Page 1
image

### Page 3
image

---
#### Note
This PR did not update the styling to Page 2 of the New Course form.

**Related Issue #670**

### CODE REVIEW

File: src/components/Form/FormFormats.js [hunk 1]
```diff
@@ -1381,13 +1404,38 @@ export default {
...
+                        name: 'courseLink',
+                        ...stringField(
+                            'Meeting Link',
+                            new_course_form.textFields
+                        ),
+                    },
+                    {
+                        name: 'googleClassCode',
+                        ...stringField(
+                            'GClassromm Enrollment Code',
```
Comment (dhuangg) [inline src/components/Form/FormFormats.js:1426]: Typo here

File: src/components/Form/FormFormats.js [hunk 2]
```diff
@@ -1648,16 +1701,47 @@ export default {
...
+                            const cachedCourseManagementCourses = cache.readQuery(
+                                {
+                                    query: GET_COURSES_BY_ACCOUNT_ID,
+                                    variables: {
+                                        accountId: '',
+                                    },
+                                }
+                            );
+
+                            if (cachedCourseManagementCourses !== null) {
```
Comment (dhuangg) [inline src/components/Form/FormFormats.js:1731]: oh wow. i thought it would usually break on cache.readQuery. this works though?

**Review by dhuangg — APPROVED** [review]
1 small typo fix, and we're good to go! I didn't realize that Katie moved the "Back" button on the form to be after the "Cancel" button which feels strange...

File: src/components/Form/FormFormats.js (hunk 2 above)
Comment (andrewjknapp) [reply src/components/Form/FormFormats.js:1731]: Yeah this works, if that query does not exist in the cache it returns null, so this handles that case.

**Pull request merged**